docs/kopia_troubleshooting.md

```diff
@@ -123,4 +114,184 @@ Users may also find the following type of errors in the logs while maintenance i
   * `Error getting a backup store`
   * `BackupStorageLocation is invalid`
 
-     
\ No newline at end of file
+## Kopia Repository Maintenance
+
+* repository maintenance commands:
+```
+kopia maintenance info
+kopia maintenance run 
```

Comment:
Before running maintenance:
 1. Ensure there are no running Backups nor Backup will ran during maintenance task
 2. Take ownership of maintenance by:
 ```shell
   # Save current ownership to revert it back later
   $ kopia maintenance info | grep Owner
   Owner: default@default

   # Set ownership to the current user in the pod
   $ kopia maintenance set --owner=me
   Setting maintenance owner to root@oadp-mustgather-pod

   #########
   # Perform maintenance tasks
   #########

   # Revert back to the previous owner, so OADP can function
   $ kopia maintenance set --owner=default@default
   Setting maintenance owner to default@default

 ```

Reply:
@mpryc thanks for adding this.. One of the times I execute maint I had to update the user.  Other time I didn't.    If the user on the must-gather container is not setup for kopia to make it EASY, I do wonder if this is enough of a use case to ship the kopia container on it's own. WDYT?   Until then I have incorporated these instructions into the doc.  Please check it out.